### PR TITLE
improve after dry run through

### PR DESCRIPTION
**Pull request by MarkCheshire**
@mayorova hope this is fine. Any tips for the whitespace changes on some of the newlines?

### CODE REVIEW

File: doc/openshift-guide.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ where `ec2-54-321-67-89.compute-1.amazonaws.com` is the Public Domain, and `54.3
 
 ## Tutorial Steps
 
-### Create your 3scale API Gateway using a template
+### Create your APIcast Gateway using a template
 
 1. Login into OpenShift using the `oc` command from the OpenShift Client tools you downloaded and installed in previous step. The default login credentials are _username = "admin"_ and _password = "admin"_
```
Comment (mayorova) [inline doc/openshift-guide.md]: Here is where the problem lies...
Here it is suggested to log in as admin, and later it says you should log into the console as "developer".
I followed the tutorial as "admin", and missed that "developer" part. But I'll change it to use developer throughout the tutorial.

File: doc/openshift-guide.md [hunk 2]
```diff
@@ -177,48 +177,54 @@ where `ec2-54-321-67-89.compute-1.amazonaws.com` is the Public Domain, and `54.3
...
-### Deploying 3scale API Gateway
+### Deploying APIcast Gateway
 
 1. Open the web console for your OpenShift cluster in your browser: https://OPENSHIFT-SERVER-IP:8443/console/
 
  You should see the login screen:
  <img src="https://support.3scale.net/images/screenshots/guides-openshift-login-screen.png" alt="OpenShift Login Screen">
 
-2. Login using your credentials created or obtained in the _Setup OpenShift_ section above.
+2. Login using your _developer_ credentials created or obtained in the _Setup OpenShift_ section above.
```
Comment (mayorova) [inline doc/openshift-guide.md]: Ah, actually it wasn't specified which user to use before... :) That's the problem.

File: doc/openshift-guide.md [hunk 3]
```diff
@@ -177,48 +177,54 @@ where `ec2-54-321-67-89.compute-1.amazonaws.com` is the Public Domain, and `54.3
...
 
  You will see a list of projects, including the _"gateway"_ project you created from the command line above.
 
  <img src="https://support-preview.3scale.net/images/screenshots/guides-openshift-project-list-after.png" alt="Openshift Projects" >
 
+ If you do not see your gateyway project, assign it to to this user:
```
Comment (mayorova) [inline doc/openshift-guide.md]: After changing everything to use "developer" user, this shouldn't bee needed.

**Comment by MarkCheshire** [comment]
OC leads you along the path to login to CLI with admin credentials and to console with developer credentials. We can with go along with this flow and include the comment on policy assignment. Or if it is bad practice to use admin credentials as @mikz suggested then we set users on a better path. Let me know how you vote and I amend

**Comment by mikz** [comment]
@MarkCheshire `oc cluster up` does not really lead you anywhere:

```
   The server is accessible via web console at:
       https://172.31.7.248:8443

   The metrics service is available at:
       https://metrics-openshift-infra.35.156.158.21.xip.io

   You are logged in as:
       User:     developer
       Password: developer

   To login as administrator:
       oc login -u system:admin

``` 

It actually logs you in as `developer`.

**Comment by mayorova** [comment]
Right, we'd just need to remove point 1 [here](https://github.com/MarkCheshire/apicast/blob/86bbecbb418e9cddfbb39c3e8438e833aa2d6d15/doc/openshift-guide.md#create-your-apicast-gateway-using-a-template), it's not needed, as the user is already logged in as developer at this moment.

**Comment by MarkCheshire** [comment]
Hopefully the second commit covers all the points - making sure that the entire flow is done with developer creds

**Pull request merged**